examples/commands/reload.js

```diff
@@ -0,0 +1,43 @@
+/*
+THIS IS JUST THE COMMAND IT SELF, IF YOU USE THIS EXACTLY THEN YOU WILL NEED A WAY TO LOAD THE FILE LIKE THE FOLLOWING HERE:
+https://solaris.codes/erelajs/guides/moreCommands.html#before-you-start
+
+YOU ALSO NEED TO INITIATE THE MANAGER AS SHOWN HERE:
+https://solaris.codes/erelajs/guides/basics.html#first-start
+
+Or copy the code inside the run function as its simply the message and arguments.
+*/
+
+
+const Discord = require("discord.js")
+
+module.exports = {
+    name: "reload",
+    run: async (client, message, args) => {
```

Comment:
Fix the function argument , `client` is a parameter here instead use `message.client`